src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -254,25 +253,25 @@ Amplify.configure(awsconfig);
 ```
 
 This will bootstrap your app to use Amplify with the deployed backend resources.
-
+ 
```

Comment:
Is there anything more we want to add to tie-in to real-world use and application of the topic discussed for the Recap below?

Reply:
no - I think this is fine for now.